vault/resource_identity_group_member_group_ids.go

```diff
@@ -0,0 +1,149 @@
+package vault
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/hashicorp/terraform-provider-vault/internal/consts"
+	"github.com/hashicorp/terraform-provider-vault/internal/identity/group"
+	"github.com/hashicorp/terraform-provider-vault/internal/provider"
+)
+
+func identityGroupMemberGroupIdsResource() *schema.Resource {
+	return &schema.Resource{
+		Create: identityGroupMemberGroupIdsUpdate,
+		Update: identityGroupMemberGroupIdsUpdate,
+		Read:   ReadWrapper(identityGroupMemberGroupIdsRead),
+		Delete: identityGroupMemberGroupIdsDelete,
+
+		Schema: map[string]*schema.Schema{
+			consts.FieldMemberGroupIDs: {
+				Type:     schema.TypeSet,
+				Optional: true,
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+				},
+				Description: "Group IDs to be assigned as group members.",
+			},
+			consts.FieldExclusive: {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  true,
+				Description: `Should the resource manage member group ids
+exclusively? Beware of race conditions when disabling exclusive management`,
+			},
+			consts.FieldGroupID: {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: "ID of the group.",
+			},
+		},
+	}
+}
+
+func identityGroupMemberGroupIdsUpdate(d *schema.ResourceData, meta interface{}) error {
```

Comment:
Aside from the field names is there any other difference between this func and its ancestor: `identityGroupMemberEntityIdsUpdate()` ?  If not then you could probably  add some getter's  that return the underlying CRUD func based on the field name provided?

Reply:
Good point. I factored out common CRUD functions that execute appropriately based on the field name provided. Thanks for the callout! 😄